Lib/fontParts/base/normalizers.py

```diff
@@ -290,12 +290,16 @@ def normalizeGlyphUnicodes(value):
     * **value** must be a ``list``.
     * **value** items must normalize as glyph unicodes with
       :func:`normalizeGlyphUnicode`.
-    * Returned value will be a ``list`` of ints.
+    * Returned value will be a ``list`` of unique ints.
     """
     if not isinstance(value, list):
         raise TypeError("Glyph unicodes must be a list, not %s."
                         % type(value).__name__)
-    return [normalizeGlyphUnicode(v) for v in value]
+    values = [normalizeGlyphUnicode(v) for v in value]
+    # Remove duplicate values while preserving their order
+    seen = set()
+    seen_add = seen.add
+    return [v for v in values if not (v in seen or seen_add(v))]
```

Comment:
Should this be changing to a tuple, instead of here: https://github.com/robofab-developers/fontParts/blob/8fea4e25e174b0a8e53d4b2717769e0a466eddb4/Lib/fontParts/base/glyph.py#L209 (@typesupply?)